src/cbc_sdk/platform/processes.py

```diff
@@ -316,6 +316,20 @@ def process_pids(self):
         else:
             return None
 
+    def deobfuscate_cmdline(self):
+        """
+        Deobfuscates the command line of the process and returns the deobfuscated result.
+
+        Required Permissions:
+            script.deobfuscation(EXECUTE)
+
+        Returns:
+             dict: A dict containing information about the obfuscated command line, including the deobfuscated result.
+        """
+        body = {"input": self.process_cmdline}
```

Comment:
process_cmdline is a list, so think this needs to be
body = {"input": self.process_cmdline[0]}

Reply:
Yes, I see this. I'll change it, which also involves changing the test code.